File: Exercises for Programmers - 57 challenges/ch.01/2017.04.26.r
bill <- readline(prompt = "What is the bill? ")
bill <- as.integer(bill)
tipRate <- readline(prompt = "What is the tip percentage? ")
tipRate <- as.integer(tipRate)

total <- bill + bill * tipRate / 100

cat(sprintf("The tip is %d%%.", tipRate), "\n")
cat(sprintf("The total is $%.2f.", total))

File: Exercises for Programmers - 57 challenges/ch.01/2017.04.26.py
bill = float(input("What is the bill? "))
tipRate = int(input("What is the tip percentage? "))

total = bill + bill * tipRate / 100

print "The tip is " + "{:d}".format(tipRate) + "%."
print "The total is $" + "{:.2f}".format(total) + "."
